smoke/README.md

```diff
@@ -0,0 +1,105 @@
+# QA for OpenShift Shared Resource CSI Driver
+This is a Behavior-driven development (or BDD) test framework leveraging `behave` for `python`. BDD is an agile software development technique that encourages collaboration between developers, QA and non-technical or business participants in a software project.Behave uses tests written in a natural language style, backed up by Python code.
+
+We are using this framework to achieve the QA verification of any Jira Stories development takes on for the OpenShift Shared Resource CSI Driver, as well as any regression test suites the team wants to cultivate for the OpenShift Shared Resource CSI Driver.
+
+Hence, the reader should consider these tests the 'next phase' of testing after the [golang based tests](https://github.com/openshift/csi-driver-shared-resource/tree/master/test/e2e) that run in OpenShift CI prow jobs as part of merging pull requests with code changes.
+
+
+# Run smoke test on local machine
+<pre>
+- oc login [cluster-server] -u username -p password --kubeconfig=kubeconfig
+- export KUBECONFIG=kubeconfig
+- make smoke
+</pre>
+# Add Test Cases
+Behave operates on directories containing:
+
+- feature files written by your Business Analyst / Sponsor / whoever with your behaviour scenarios in it.
+- a “steps” directory with Python step implementations for the scenarios.
+
+The minimum requirement for a features directory is:
+<pre>
+features/
+features/everything.feature
+features/steps/
+features/steps/steps.py
+</pre>
+A more complex directory might look like:
+<pre>
+features/
+features/signup.feature
+features/login.feature
+features/account_details.feature
+features/environment.py
+features/steps/
+features/steps/website.py
+features/steps/utils.py
+</pre>
+A feature file has a natural language format describing a feature or part of a feature with representative examples of expected outcomes. They’re plain-text (encoded in UTF-8) and look something like:
+<pre>
+Feature: Fight or flight
+  In order to increase the ninja survival rate,
+  As a ninja commander
+  I want my ninjas to decide whether to take on an
+  opponent based on their skill levels
+
+  Scenario: Weaker opponent
+    Given the ninja has a third level black-belt
+     When attacked by a samurai
+     Then the ninja should engage the opponent
+
+  Scenario: Stronger opponent
+    Given the ninja has a third level black-belt
+     When attacked by Chuck Norris
+     Then the ninja should run for his life
+</pre>
```

Comment:
```suggestion
\```
Feature: Fight or flight
  In order to increase the ninja survival rate,
  As a ninja commander
  I want my ninjas to decide whether to take on an
  opponent based on their skill levels

  Scenario: Weaker opponent
    Given the ninja has a third level black-belt
     When attacked by a samurai
     Then the ninja should engage the opponent

  Scenario: Stronger opponent
    Given the ninja has a third level black-belt
     When attacked by Chuck Norris
     Then the ninja should run for his life
\```
```